### DISCUSSION TITLE
Can I connect all outputs to one controller?

### DISCUSSION THREAD

**Discussion by LEOsMIND** [darthcloud/BlueRetro#689] [Q&A] [answered]
Hi,
I am looking for a way to control 4 Gamecubes with one controller. I got the BlueRetro, but of course it emulates 4 different controllers.  

So my idea was to connect the Adapter with bluetooth to my PC, simulate 4 controllers and connect it to the BlueRetro. Then I can control the 4 Virtual Controllers with one. I know that sounds way too complicated, but it's my only idea as a newbie.

So is there a way to connect the BlueRetro adapter to my PC? In Windows 11 it won't get a connection.
Or is there even a simpler solution to control 4 Gamecubes with one controller?

**Comment by darthcloud** [comment, marked as answer]
Yes you can and its simpler than what you suggest, see details here:
https://github.com/darthcloud/BlueRetro/discussions/675

**Reply by LEOsMIND** [reply]
thank you so much!
How do I make sure to provide power through only one Gamecube?
